docker-compose.yml

```diff
@@ -55,7 +55,8 @@
 
 version: '3.5'
 
-x-ccache: &ccache
+x-ccache:
+  &ccache
```

Comment:
Could you revert style changes?

Reply:
Yes, absolutely — did not intend for those to happen!